cmd/generic/cmd/jsontest.go

```diff
@@ -224,7 +224,7 @@ func runSSHCmd(_ *cobra.Command, args []string) {
 	// SSH shell creation.
 	goExpectSpawner := interactive.NewGoExpectSpawner()
 	var spawnContext interactive.Spawner = goExpectSpawner
-	context, err := interactive.SpawnSSH(&spawnContext, user, host, (*tester).Timeout(), interactive.Verbose(true))
+	context, err := interactive.SpawnSSH(&spawnContext, user, host, (*tester).Timeout(), interactive.Verbose(true), interactive.SendTimeout((*tester).Timeout()))
 	if err != nil {
 		fatalError("could not create the ssh expecter", err, testExpecterCreationFailedExitCode)
```

Comment:
Line 254: should runOcCmd use the extra sendtimeout option? 
oc, ch, err := interactive.SpawnOc(&spawnContext, pod, container, namespace, (*tester).Timeout(), interactive.Verbose(true))

Reply:
fixed.